actors/evm/src/interpreter/precompiles/parameter.rs

```diff
@@ -84,12 +84,22 @@ impl_param_int!(u16 i16 u32 i32 u64 i64);
 /// Provides a nice API interface for reading Parameters from input. This API treats the input as if
 /// it is followed by infinite zeros.
 pub(super) struct ParameterReader<'a> {
+    full: &'a [u8],
     slice: &'a [u8],
 }
```

Comment:
we don't need this ATM im pretty sure, but being generic over 2 lifetimes, `where 'a: 'b` may improve usability.